backend/src/zac/reports/api/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from ..models import Report
+
+
+class ReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Report
+        fields = (
+            "id",
+            "name",
+        )
+
+
+class ReportDownloadSerializer(serializers.Serializer):
+    identificatie = serializers.CharField()
+    omschrijving = serializers.CharField()
+    startdatum = serializers.DateField()
+    status = serializers.CharField()
+    zaaktype_omschrijving = serializers.CharField()
```

Comment:
```suggestion
    zaaktype_omschrijving = serializers.CharField(source="zaaktype.omschrijving")
```
-> this way you can pass _objects_ in the view rather than doing the serialization manually.

Reply:
Thanks. Keep making this mistake... :(